src/AutoRest.CSharp/Common/Output/Builders/BuilderHelpers.cs

```diff
@@ -289,11 +296,7 @@ public static FormattableString CreateDerivedTypesDescription(CSharpType type)
         }
 
         public static string CreateDescription(this Schema schema)
-        {
```

Comment:
This causes [Description update](https://github.com/Azure/azure-sdk-for-net/pull/44148/files#diff-327d085438f0bbac252b734c863541084d36954d7ed0966205310e77e5c5b1deL14)